tests/testExecutables.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+import lsst.utils.tests
+
+
+class UtilsBinaryTester(lsst.utils.tests.ExecutablesTestCase):
+    pass
+
+EXECUTABLES = None
+UtilsBinaryTester.create_executable_tests(__file__, EXECUTABLES)
```

Comment:
I think the `exes` argument should be defaulted to `None` so it doesn't need to be specified explicitly.  Why create additional boilerplate?  In fact, can't the `create_executable_tests` call be folded in to the `class` definition somehow, e.g., in the `__init__` or through a metaclass?